.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: python
-
 python:
-  - "3.7"
-
+- '3.7'
 install:
-  - pip install -r requirements.txt
-
+- pip install -r requirements.txt
+before_script:
+- export APP_SETTINGS="config.DevelopmentConfig"
+- export DATABASE_URL="postgresql://localhost/language_learner_dev"
 script:
-  - pytest
+- python3 manage.py db upgrade
```

Comment:
It didn't like this command in the build

Reply:
I'll remove the line and push it again to heroku-staging